__tests__/integration/query.test.ts

```diff
@@ -62,7 +69,7 @@ describe("query", () => {
   it("Can query with arguments", async () => {
     const str = "taco";
     const result = await client.query(fql`${str}.length`);
-    expect(result.data).toEqual(4);
```

Comment:
why did we change all these? just curious

Reply:
A nit pick. `.toBe` is supposed to be for comparing literal values and `.toEqual` is for comparing objects.

Reply:
my understanding is toBe compares refernces (addresses in memory) and toEqual compares values.

I don't think there's really any reason for us to care about the references in memory of these types of things.

Reply:
`toBe` might compare the reference for Objects, but the docs say it is for asserting a value is what you expect.
![image](https://user-images.githubusercontent.com/7998974/231837375-a6dd3351-e432-4ec6-8678-3b400cc01910.png)